File: 1038.py
a, b = input().split(' ')
b = int(b)
if a == '1':
    a = 4.0
if a == '2':
    a = 4.5
if a == '3':
    a = 5.0
if a == '4':
    a = 2.0
if a == '5':
    a = 1.5
print(f'Total: R$ {a*b:.2f}')

File: 1038.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = lines.shift().split(' ');
var a = n[0]; var b = Number(n[1]);
if (a === '1'){
    var a = 4.00}
if (a === '2'){
    var a = 4.50}
if (a === '3'){
    var a = 5.00}
if (a === '4'){
    var a = 2.00}
if (a === '5'){
    var a = 1.50}
console.log(`Total: R$ ${(a*b).toFixed(2)}`);
